android/src/main/java/org/gooddollar/FaceTecModule.java

```diff
@@ -177,4 +183,29 @@ public void faceVerification(String enrollmentIdentifier,
         lastProcessor = processor;
         processor.enroll(enrollmentIdentifier, maxRetries);
     }
+
+    private FaceTecSDKStatus getSDKStatus(Activity activity) {
```

Comment:
it seems we don't need  a separate method
on IOS implementation getStatus is needed to be wrapped onto the main thread
if you don't need additional logic here, just use FaceTecSDK.getStatus(activity) everywhere